### PR TITLE
Fix Chrome 87 fieldset rendering bug

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it happened:
Screenshot 2020-11-30 at 12 57 58
Screenshot 2020-11-26 at 15 47 06
Screenshot 2020-11-25 at 11 59 41

... and now it doesn't.

Ref https://www.drupal.org/project/drupal/issues/3183425

**PR intended to be tested with API branch:** master

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.

### Test environment config

API_URI=https://master.staging.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1751 Investigate rendering in chrome 87](https://app.clickup.com/t/2549495/SALEOR-1751)

**Comment by maarcingebala** [comment]
I thought it's the fault of a beta version of Chrome but now I'm on a stable version and still getting these issues constantly. Glad it will be resolved 🙏

**Review by orzechdev — CHANGES_REQUESTED** [review]
CI fails completely

File: src/theme.ts [hunk 1]
```diff
@@ -73,7 +229,7 @@ export default (colors: IThemeColors): Theme =>
           borderRadius: 8,
           borderStyle: "solid",
           borderWidth: 1,
-          overflow: "visible"
+          overflow: "hidden"
```
Comment (orzechdev) [inline src/theme.ts:232]: Probably this is the cause of input issue - without this change, it displayed popup also outside of the card:
Zrzut ekranu 2020-12-2 o 13 23 02

**Review by orzechdev — CHANGES_REQUESTED** [review]
CI is working now, but I see there is an issue with select input:
Zrzut ekranu 2020-12-2 o 13 26 09

File: src/theme.ts (hunk 1 above)
Comment (dominik-zeglen) [reply src/theme.ts:232]: Aw damn this is part of debug code 🙄

**Review by orzechdev — APPROVED** [review]
It's weird, test env still has the old code. But tested locally and input now is displayed correctly.

**Review by mmarkusik — APPROVED** [review]

**Pull request merged**